utils/src/main/scala/uk/gov/nationalarchives/utils/TestUtils.scala

```diff
@@ -28,12 +28,12 @@ object TestUtils:
     )
 
     def createReIndexerTable(): Int =
-      sql"CREATE TABLE IF NOT EXISTS files(version int, id text, name text, fileId text, zref text, path text, fileName text, ingestDateTime numeric, sourceId text, citation text);".update.run
+      sql"CREATE TABLE IF NOT EXISTS files(version int, id text, name text, fileId text, zref text, path text, fileName text, ingestDateTime numeric, sourceId text, citation text, consignmnetRef text);".update.run
         .transact(xa)
         .unsafeRunSync()
 
     def createTable(): Int =
-      sql"CREATE TABLE IF NOT EXISTS files(version int, id text, name text, fileId text, zref text, path text, fileName text, ingestDateTime numeric, sourceId text, citation text);".update.run
+      sql"CREATE TABLE IF NOT EXISTS files(version int, id text, name text, fileId text, zref text, path text, fileName text, ingestDateTime numeric, sourceId text, citation text, consignmentRef text);".update.run
```

Comment:
This wasn't caused by anything you did but this change has highligted it. The tests are failing for me now because I have an existing database so the table doesn't get recreated. To get round this, we can drop and recreate the table for each test. You can do something like:
```scala
    def createTable(): Unit = {
      val transaction = for {
        _ <- sql"DROP TABLE IF EXISTS files".update.run
        _ <- sql"CREATE TABLE IF NOT EXISTS files(version int, id text, name text, fileId text, zref text, path text, fileName text, ingestDateTime numeric, sourceId text, citation text, consignmentRef text);".update.run
      } yield ()
      transaction.transact(xa).unsafeRunSync()
    }
```
I've tried it and it's pretty quick. Same for the `createReIndexerTable` method.

Reply:
Of course. It makes sense. Also, as we will drop the table, do we need to check for "IF NOT EXISTS" when creating